tests/transform_dialect/cpu/BUILD

```diff
@@ -35,6 +35,7 @@ iree_lit_test_suite(
         "//tools:iree-compile",
         "//tools:iree-opt",
         "//tools:iree-run-module",
+        "@llvm-project//lld",
```

Comment:
[here for threading]

Why wasn't this failing on CI?

Reply:
Good question, I just noticed it when I was trying to integrate this and testing with blaze showed this. I would have expected bazel to also flag this.

Reply:
Could be that it works in Bazel if lld is already on the user's path? So much for hermetic :-/ Would be nice to track down the cause here, but this change seems safe